src/components/AuthCard.js

```diff
@@ -108,73 +111,84 @@ function AuthCard({ setToken }) {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <Card className={classes.card}>
-        <CardContent>
-          <Typography align="center" gutterBottom variant="h5" component="h2">
-            Welcome to Jangos Island
-          </Typography>
-          <TextField
-            className={classes.withMargin}
-            {...text("username")}
-            label="Username"
-            required
-            variant="outlined"
-            fullWidth
-          />
-          <TextField
-            className={classes.withMargin}
-            {...text("password")}
-            label="Password"
-            required
-            variant="outlined"
-            fullWidth
-            type="password"
-          />
-          {isNewUser && (
+    <div>
+      <CardMedia
+        className={classes.media}
+        component="img"
+        image={require("../assets/JangosIsland.png")}
+        title="island"
+
+        // style={styles.media}
+      />
+      <form onSubmit={handleSubmit}>
+        <Card className={classes.card}>
+          <CardContent>
+            <Typography align="center" gutterBottom variant="h5" component="h2">
+              Can you escape the island?
+            </Typography>
+            <TextField
+              className={classes.withMargin}
+              {...text("username")}
+              label="Username"
+              required
+              variant="outlined"
+              fullWidth
+            />
             <TextField
               className={classes.withMargin}
-              {...text("password2")}
-              label="Repeat Password"
+              {...text("password")}
+              label="Password"
```

Comment:
This is asking the user to repeat the password for the second time to make sure the password is a match. 
```suggestion
              label="Repeat Password"
```